{
  "uid" : "3a1a68b5ef2be1cc",
  "name" : "In-place",
  "fullName" : "Connection.In-place",
  "historyId" : "ad5f1e14898f92a50fac41fda7599983",
  "time" : {
    "start" : 1606804761635,
    "stop" : 1606804790655,
    "duration" : 29020
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "failed",
  "statusMessage" : "Error: Timeout - Async callback was not invoked within the 300000 ms timeout specified by jest.setTimeout.",
  "statusTrace" : "Timeout - Async callback was not invoked within the 300000 ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 300000 ms timeout specified by jest.setTimeout.\n                    at mapper (/home/runner/work/jetpack/jetpack/node_modules/jest-config/node_modules/jest-jasmine2/build/queueRunner.js:27:45)\n                    at /home/runner/work/jetpack/jetpack/node_modules/jest-config/node_modules/jest-jasmine2/build/queueRunner.js:75:41\n                    at runMicrotasks (<anonymous>)\n                    at processTicksAndRejections (internal/process/task_queues.js:97:5)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "Error: Timeout - Async callback was not invoked within the 300000 ms timeout specified by jest.setTimeout.",
    "statusTrace" : "Timeout - Async callback was not invoked within the 300000 ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 300000 ms timeout specified by jest.setTimeout.\n                    at mapper (/home/runner/work/jetpack/jetpack/node_modules/jest-config/node_modules/jest-jasmine2/build/queueRunner.js:27:45)\n                    at /home/runner/work/jetpack/jetpack/node_modules/jest-config/node_modules/jest-jasmine2/build/queueRunner.js:75:41\n                    at runMicrotasks (<anonymous>)\n                    at processTicksAndRejections (internal/process/task_queues.js:97:5)",
    "steps" : [ {
      "name" : "Can start in-place connection",
      "time" : {
        "start" : 1606804761636,
        "stop" : 0,
        "duration" : -1606804761636
      },
      "status" : "unknown",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "6a362361660f7adf",
        "name" : "Test failed: Connection :: In-place",
        "source" : "6a362361660f7adf.png",
        "type" : "image/png",
        "size" : 4214
      } ],
      "parameters" : [ ],
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true,
      "stepsCount" : 0
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true,
    "stepsCount" : 1
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Connection"
  }, {
    "name" : "testClass",
    "value" : "Connection"
  }, {
    "name" : "testMethod",
    "value" : "In-place"
  }, {
    "name" : "package",
    "value" : "Connection"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "3a1a68b5ef2be1cc.json",
  "parameterValues" : [ ]
}